source/iface/event.rst

```diff
@@ -217,7 +364,7 @@ event profiling info.
    :lines: 5-
    :linenos:
```

Comment:
Some comments on the example:

* This two lines are missing the `std::` namespace:
  ```
  auto src = aligned_alloc(alignment, num_bytes);
  memset(src, 1, num_bytes);
  ```

* The example should have `#include <cstring>` and `#include <cstdlib>` because of the two calls above.

Reply:
Fixed in the latest commit